src/components/RentalsOverview/RentalsOverview.tsx

```diff
@@ -39,10 +42,22 @@ const RentalsOverview = ({ componentType }: RentalsOverviewProps) => {
 
   return (
     <div className={classes['c-rentals-overview']}>
-      <div className={classes['c-rentals-overview__headers']}>
+      <div
+        className={`${classes['c-rentals-overview__headers']} ${
+          isRoleAdmin()
+            ? classes['c-rentals-overview__headers-admin']
+            : classes['c-rentals-overview__headers-user']
+        }`}
+      >
         <span>Book</span>
         <span>Start date</span>
         <span>End date</span>
+        {isRoleAdmin() ? (
+          <>
+            <span>User email</span>
+            <span>Extend rent period for (days):</span>
+          </>
+        ) : null}
```

Comment:
This part should be with `&&` as I suggested, guess it was lost during rebase :)